src/Common/src/Interop/Windows/advapi32/Interop.ClearEventLog.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Advapi32
+    {
+        [DllImport(Libraries.Advapi32, CharSet = CharSet.Unicode, SetLastError = true)]
+        public static extern bool ClearEventLog(SafeHandle hEventLog, HandleRef lpctstrBackupFileName);
```

Comment:
Use the exact types of the SafeHandles that the methods expect everywhere? (ie `SafeEventLogReadHandle` or `SafeEventLogWriteHandle` instead of just `SafeHandle`)

Reply:
It expected `SafeEventLogReadHandle`.  Should I change `SafeHandle ` to `SafeEventLogReadHandle`?

Reply:
Yes I think so.

Reply:
ok. done